### PR TITLE
MAUI-849605 Updated Polar Chart UG page in .NET MAUI

### PR DESCRIPTION
**Pull request by snehakPV**
**Feature Description**

Updated UG for Polar Chart (new control).

**Analysis and Design**

Referred the Xamarin, WinUI and WPF UG
Xamarin - https://help.syncfusion.com/xamarin/charts/charttypes#polar-chart
WinUI - https://help.syncfusion.com/winui/polar-chart/overview
WPF - https://help.syncfusion.com/wpf/charts/seriestypes/radarandpolar

**Solution Description**

This UG document helps to guide the user to understand the feature.

**Updated Files**

	1. Migrate from Xamarin.Forms
	2. Axis
		○ Types
		○ Title
		○ Axis Labels
		○ Grid Lines
		○ Tick Lines
		○ Axis Line
	3.  Chart Types
		○ Polar Area Chart
		○ Polar Line Chart
	4. Appearance
	5. Data Label
	6. Tooltip
	7. Exporting
	8. Get the touch position in chart
	9. Transform axis value to pixel value and vice versa

### CODE REVIEW

File: MAUI/Polar-Charts/Axis/Axislabels.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Axis labels in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Axislabels.md:3]: title: Axis Labels in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/Axis/Axisline.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Axis line in .NET MAUI Chart
+
+## Customization
+
+The polar chart axis provides support for customizing the style of the axis line by defining the [AxisLineStyle]()property, as shown in the code snippet below.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Axisline.md:14]: The polar chart axis provides support for customizing the style of the axis line by defining the [AxisLineStyle]()property, as shown in the code sample below.

File: MAUI/Polar-Charts/Axis/Gridlines.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: Axis grid lines in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Gridlines.md:3]: title: Axis Grid Lines in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/Axis/Overview.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: About Axis in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Overview.md:3]: title: About Axis in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/Axis/Overview.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+Polar chart supports the following types of chart axis.
+
+* Category
+* Numeric
+* Date time
+
+## Key features
+
+* Axis supports different types of axes such as Category, Numeric, and Date Time.
+* It allows you to customize the chart axis line, including grid lines and tick lines. You can also customize the style of the axis line by defining the [AxisLineStyle]().
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Overview.md:23]: * It allows you to customize the chart axis line, including grid lines, and tick lines. You can also customize the style of the axis line by defining the [AxisLineStyle]().

File: MAUI/Polar-Charts/Axis/Ticklines.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: Axis Tick Line in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Ticklines.md:3]: title: Axis Tick Line in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/Axis/Ticklines.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+};
+chart.SecondaryAxis.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Customization
+
+Both major and minor tick lines can be customized by using the [MajorTickStyle]() and [MinorTickStyle]() properties respectively. They provide options to change the [StrokeWidth](), [TickSize](), [Stroke]() of tick lines. By default minor tick lines will not be visible.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Ticklines.md:55]: Both major and minor tick lines can be customized using the [MajorTickStyle]() and [MinorTickStyle]() properties respectively. They provide options to change the [StrokeWidth](), [TickSize](), [Stroke]() of tick lines. By default, minor tick lines will not be visible.

File: MAUI/Polar-Charts/Axis/Title.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: Title for axis in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Title.md:3]: title: Title for axis in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/Axis/Types.md [hunk 9]
```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: Axis types in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Types.md:3]: title: Axis types in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/Axis/Types.md [hunk 10]
```diff
@@ -0,0 +1,340 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![DateTimeAxis support in MAUI Chart](Axis_Images/MAUI_datetime_axis.png)
+
+### Interval
+
+In [DateTimeAxis](), intervals can be customized by using the [Interval]() and [IntervalType]() properties. For example, setting [Interval]() as 6 and [IntervalType]() as [Months]() will consider 6 months as interval.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Types.md:225]: In [DateTimeAxis](), intervals can be customized using the [Interval]() and [IntervalType]() properties. For example, setting [Interval]() as 6 and [IntervalType]() as [Months]() will consider 6 months as interval.

File: MAUI/Polar-Charts/Axis/Types.md [hunk 11]
```diff
@@ -0,0 +1,340 @@
...
+};
+chart.PrimaryAxis.Add(primaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Inversed
+
+Axis can be inverted by using the [IsInversed]() property. The default value of this property is `False`.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Axis/Types.md:297]: Axis can be inverted using the [IsInversed]() property. The default value of this property is `False`.

File: MAUI/Polar-Charts/DataLabels.md [hunk 12]
```diff
@@ -0,0 +1,257 @@
+---
+layout: post
+title: Data labels in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/DataLabels.md:3]: title: Data Labels in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/DataLabels.md [hunk 13]
```diff
@@ -0,0 +1,257 @@
...
+
+chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Data label in MAUI chart](DataLabel_images/MAUI_polar_line_datalabel.png)
+
+Data labels can be customized by using the [DataLabelSettings]() property of chart series. To customize them, you need to create an instance of [PolarDataLabelSettings]() and set it to the [DataLabelSettings]() property. The following properties available in [PolarDataLabelSettings]() can be used to customize the data labels.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/DataLabels.md:56]: Data labels can be customized using the [DataLabelSettings]() property of chart series. To customize them, you need to create an instance of [PolarDataLabelSettings]() and set it to the [DataLabelSettings]() property. The following properties available in [PolarDataLabelSettings]() can be used to customize the data labels.

File: MAUI/Polar-Charts/DataLabels.md [hunk 14]
```diff
@@ -0,0 +1,257 @@
...
+
+* [BarAlignment]() - Gets or sets the data label alignment top, middle or bottom.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/DataLabels.md:58]: * [BarAlignment]() - Gets or sets the data label alignment top, middle, or bottom.

File: MAUI/Polar-Charts/DataLabels.md [hunk 15]
```diff
@@ -0,0 +1,257 @@
...
+* [LabelPlacement]() - Gets or sets the data label position inside, outside or default.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/DataLabels.md:59]: * [LabelPlacement]() - Gets or sets the data label position inside, outside, or default.

File: MAUI/Polar-Charts/DataLabels.md [hunk 16]
```diff
@@ -0,0 +1,257 @@
...
+* [LabelStyle]() - Gets or sets the options for customizing the data labels. 
+* [UseSeriesPalette]() - Gets or sets a value indicating whether the data label should reflect the series interior.
+
+## Data Label Alignment
+
+The alignment of data labels inside the series is defined by using the [BarAlignment]() property.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/DataLabels.md:65]: The alignment of data labels inside the series is defined using the [BarAlignment]() property.

File: MAUI/Polar-Charts/DataLabels.md [hunk 17]
```diff
@@ -0,0 +1,257 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## LabelPlacement
+
+In addition to the alignment options mentioned above, the Chart provides additional customization options for positioning the data labels.
+
+The [LabelPlacement]() property is used to position the data labels at [Center](), [Inner]() and [Outer]() positions relative to the actual data point position. By default, labels are positioned based on the series types for better readability.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/DataLabels.md:108]: The [LabelPlacement]() property is used to position the data labels at [Center](), [Inner](), and [Outer]() positions relative to the actual data point position. By default, labels are positioned based on the series types for better readability.

File: MAUI/Polar-Charts/DataLabels.md [hunk 18]
```diff
@@ -0,0 +1,257 @@
...
+    chart.Series.Add(series);
+    this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+## LabelTemplate
+
+The [SfPolarChart]() provides support to customize the appearance of the datalabel by using the [LabelTemplate]() property.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/DataLabels.md:191]: The [SfPolarChart]() provides support to customize the appearance of the datalabel using the [LabelTemplate]() property.

File: MAUI/Polar-Charts/Exporting.md [hunk 19]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Exporting in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Exporting.md:3]: title: Exporting in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/Exporting.md [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+
+# Exporting in .NET MAUI Chart
+
+## Export as an image
+
+You can export the chart view as an image in the desired file format using the [SaveAsImage]() method of [SfPolarChart](). The supported image formats are **JPEG and PNG**. By default, if you don't mention any image format with the filename, the chart view will be exported as an image in the PNG format.
+
+N> The chart view can be exported as an image only when the chart view is added to the visual tree.
+
+The following code snippet demonstrates the usage of this method:
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Exporting.md:18]: The following code sample demonstrates the usage of this method:

File: MAUI/Polar-Charts/Exporting.md [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+T> We can change the image formats in the above code by changing their extensions to .jpg or .png.
+
+The exported image will be saved in the different location across the platforms.
+
+**Windows Phone, Android and MAC** – The image will be saved inside the 'Pictures' directory of the file system.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Exporting.md:38]: **Windows Phone, Android, and MAC** – The image will be saved inside the 'Pictures' directory of the file system.

File: MAUI/Polar-Charts/Exporting.md [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+
+{% endtabs %}
+
+## Get the stream of Chart
+
+The [GetStreamAsync]() method of [SfPolarChart]() is used to asynchronously get the chart view as a stream in the desired ImageFileFormat. The output stream can be passed as an input to other components that accept streams, such as PDF, Excel, and Word. The supported image file formats are **JPEG and PNG**.
+
+N> The charts stream can only be rendered when the chart view is added to the visual tree.
+
+The following code snippet demonstrates the usage of this method:
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Exporting.md:71]: The following code sample demonstrates the usage of this method:

File: MAUI/Polar-Charts/Get-the-touch-position-in-chart.md [hunk 23]
```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Get the touch position in Syncfusion SfPolarChart
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/Get-the-touch-position-in-chart.md:3]: title: Get the Touch Position in Syncfusion SfPolarChart

File: MAUI/Polar-Charts/PolarArea.md [hunk 24]
```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: Polar Area Chart in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/PolarArea.md:3]: title: Polar Area Chart in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/PolarArea.md [hunk 25]
```diff
@@ -0,0 +1,241 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Polar area chart type in MAUI Chart](Chart-types_images/MAUI_polar_area_chart.png)
+
+## Grid line Type
+
+The [GridLineType]() property used to modify the rendering style of axis grid lines. The default [GridLineType]() is [Circle](). Switching to the [Polygon]() grid line type transforms the polar chart appearance, resembling a spider or web chart.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/PolarArea.md:64]: The [GridLineType]() property used to modify the rendering style of axis grid lines. The default [GridLineType]() is [Circle](). Switching to the [Polygon]() grid line type transforms the polar chart appearance, resembling a spider, or web chart.

File: MAUI/Polar-Charts/PolarArea.md [hunk 26]
```diff
@@ -0,0 +1,241 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Polar gridline type in MAUI Chart](Chart-types_images/MAUI_polar_area_gridline.png)
+
+## Closed Series
+
+The [IsClosed]() property used to render the series with or without closed path. The default value of [IsClosed]() is true.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/PolarArea.md:103]: The [IsClosed]() property is used to render the series with or without closed path. The default value of [IsClosed]() is true.

File: MAUI/Polar-Charts/PolarLine.md [hunk 27]
```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+title: Polar Line Chart in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/PolarLine.md:3]: title: Polar Line Chart in .NET MAUI Chart Control | Syncfusion

File: MAUI/Polar-Charts/PolarLine.md [hunk 28]
```diff
@@ -0,0 +1,242 @@
...
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Polar Line Chart in .NET MAUI Chart
+
+## Polar Line Chart
+
+To display an line series in a polar chart, instantiate the [PolarLineSeries]() and include it in the [Series]() collection property of [SfPolarChart]().
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/PolarLine.md:14]: To display a line series in a polar chart, instantiate the [PolarLineSeries]() and include it in the [Series]() collection property of [SfPolarChart]().

File: MAUI/Polar-Charts/PolarLine.md [hunk 29]
```diff
@@ -0,0 +1,242 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Polar gridline type in MAUI Chart](Chart-types_images/MAUI_polar_line_gridline.png)
+
+## Closed Series
+
+The [IsClosed]() property used to render the series with or without closed path. The default value of [IsClosed]() is true.
```
Comment (florenceliliana) [inline MAUI/Polar-Charts/PolarLine.md:103]: The [IsClosed]() property is used to render the series with or without closed path. The default value of [IsClosed]() is true.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

**Review by Saravanan-Madhesh — APPROVED** [review]
Changes fine

**Pull request merged**